jfxtras-controls/src/main/java/jfxtras/internal/scene/control/skin/CalendarTextFieldSkin.java

```diff
@@ -365,7 +389,7 @@ public String getUserAgentStylesheet() {
 		};
 		lBorderPane.getStyleClass().add(this.getClass().getSimpleName() + "_popup");
 		lBorderPane.setCenter(calendarPicker);
-		calendarPicker.showTimeProperty().set( getSkinnable().getShowTime() );
+		calendarPicker.showTimeProperty().bind(getSkinnable().showTimeProperty());
```

Comment:
I did not know about memory leaks on binding. 

I think this is safe for JavaFX8. According to the JavaDocs http://docs.oracle.com/javase/8/javafx/api/javafx/beans/binding/Binding.html#dispose-- this has been addressed.
It states
> All bindings in our implementation use instances of WeakInvalidationListener, which means usually a binding does not need to be disposed. But if you plan to use your application in environments that do not support WeakReferences you have to dispose unused Bindings to avoid memory leaks.

But adding change listeners still can cause memory leaks in JavaFX8 (http://docs.oracle.com/javase/8/javafx/api/javafx/beans/value/ObservableValue.html#addListener-javafx.beans.value.ChangeListener-)